docs/plugins.md

```diff
@@ -69,253 +61,25 @@ file, you can write the plugin inline into the config:
- *     for the promise to resolve before continuing.  If the promise is
- *     rejected, a failed assertion is added to the test results.
- */
-exports.onPageStable = function() {};
-
-/**
- * Between every webdriver action, Protractor calls browser.waitForAngular() to
- * make sure that Angular has no outstanding $http or $timeout calls.
- * You can use waitForPromise() to have Protractor additionally wait for your
- * custom promise to be resolved inside of browser.waitForAngular().
- *
- * @this {Object} bound to module.exports
- *
- * @throws {*} If this function throws an error, a failed assertion is added to
- *     the test results.
- *
- * @return {q.Promise=} Can return a promise, in which case protractor will wait
- *     for the promise to resolve before continuing.  If the promise is
- *     rejected, a failed assertion is added to the test results, and protractor
- *     will continue onto the next command.  If nothing is returned or something
- *     other than a promise is returned, protractor will continue onto the next
- *     command.
- */
-exports.waitForPromise = function() {};
-
-/**
- * Between every webdriver action, Protractor calls browser.waitForAngular() to
- * make sure that Angular has no outstanding $http or $timeout calls.
- * You can use waitForCondition() to have Protractor additionally wait for your
- * custom condition to be truthy.
- *
- * @this {Object} bound to module.exports
- *
- * @throws {*} If this function throws an error, a failed assertion is added to
- *     the test results.
- *
- * @return {q.Promise<boolean>|boolean} If truthy, Protractor will continue onto
- *     the next command.  If falsy, webdriver will continuously re-run this
- *     function until it is truthy.  If a rejected promise is returned, a failed
- *     assertion is added to the test results, and protractor will continue onto
- *     the next command.
- */
-exports.waitForCondition = function() {};
-
-/**
- * Used when reporting results.
- *
- * If you do not specify this property, it will be filled in with something
- * reasonable (e.g. the plugin's path)
- *
- * @type {string}
- */
-exports.name = '';
-
-
-/**
- * Used to turn off default checks for angular stability
- *
- * Normally Protractor waits for all $timeout and $http calls to be processed
- * before executing the next command.  This can be disabled using
- * browser.ignoreSynchronization, but that will also disable any
- * <Plugin>.waitForPromise or <Plugin>.waitForCondition checks.  If you want to
- * disable synchronization with angular, but leave in tact any custom plugin
- * synchronization, this is the option for you.
- *
- * This is used by users who want to replace Protractor's synchronization code
- * with their own.
- *
- * @type {boolean}
- */
-exports.skipAngularStability
-```
-
-Each of these exported properties are optional.
+Plugins are node modules who's `module.exports` implement the `ProtractorPlugin`
```

Comment:
maybe "Plugins are a node module that export an object implementing the `ProtractorPlugin` interface"?